src/main/java/it/gov/pagopa/reminder/service/ReminderServiceImpl.java

```diff
@@ -137,14 +129,19 @@ public void getMessageToNotify() {
 		log.info("paidMessageToNotify: {}",paidMessageToNotify.size());
```

Comment:
why `paidMessageToNotify` ? Are we processing only payment messages here?

Reply:
Yes, because that query retrieves the payment messages that have already been read.
After that, this data is merged with the previous list, where both generic and payment messages to be read are retrieved.